secator/tasks/whois.py

```
@@ -0,0 +1,42 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)